apps/ecosystem/src/ecosystem.service.ts

```diff
@@ -176,4 +177,18 @@ export class EcosystemService {
     return isEmailSent;
   }
 
+  async getInvitationsByEcosystemId(ecosystemId: string,
+     pageNumber: number,
+     pageSize: number,
+     search: string,
+     userId: string
+     ): Promise<object> {
+    try {
+      const ecosystemInvitations = await this.ecosystemRepository.getInvitationsByEcosystemId(ecosystemId, pageNumber, pageSize, userId, search);
```

Comment:
Instead of destructuring the object in `service` file we should restructuring in repository

Reply:
Resolved